PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -269,6 +269,10 @@ class PsfPowershellScriptRunner
 					SSWrapper = file.path();
 				}
 			}
+			if (!std::filesystem::exists(SSWrapper))
+			{
+				MessageBoxEx(NULL, L"'StartingScriptWrapper.ps1' was not found. This is required to run the PowerShell scripts through PSF. Kindly add it under the package root folder and re-install the application.", L"MSIX Packet Support Framework", MB_OK | MB_ICONWARNING, 0);

```

Comment:
is StartingScriptWrapper.ps1 file needs to be in root folder?

Reply:
Changed the message accordingly.